docs/reference/query-dsl/filters/geo-polygon-filter.asciidoc

```diff
@@ -27,6 +27,29 @@ points. Here is an example:
 --------------------------------------------------
 
 [float]
+==== Filter Options
+
+[cols="<,<",options="header",]
+|=======================================================================
+|Option |Description
+|`_name` |Optional name field to identify the filter
+
+|`_cache` |Set to `true` to cache the *result* of the filter.
+Defaults to `false`. See <<Caching,Caching>> below.
+
+|`_cache_key` |Associate a unique custom cache key to use instead of
+the actual filter.
+
+|`validate` |Set to `false` to accept geo points with invalid latitude or
+longitude (default is `true`). *Note*: Validation only works when
+normalization has been disabled. This option will be deprecated and removed
```

Comment:
"will be deprecated"? Should this already be marked deprecated? I don't think we need to explicitly say "removed in upcoming release" because this is implied by deprecation?